packages/transfer/src/useCheck.ts

```diff
@@ -17,7 +17,7 @@ export const useCheck = (props: TransferPanelProps, panelState: TransferPanelSta
       if (typeof props.filterMethod === 'function') {
         return props.filterMethod(panelState.query, item)
       } else {
-        const label = item[labelProp.value] || item[keyProp.value].toString()
+        const label = item[labelProp.value] || item[keyProp.value].toString && item[keyProp.value].toString() || item[keyProp.value]
```

Comment:
Why not `item[keyProp.value].toString?.()`?

Reply:
Because the `key` of `item` may be a `number`, the type of `number` does't have the method of `toString`.

Reply:
I did something wrong. It was a demo that was wrong,  the reason is `item[keyProp.value]` might be `undefined`. I will correct it later.